.github/workflows/pre-submit.e2e.generic.default.yml

```diff
@@ -39,3 +39,33 @@ jobs:
           BINARY: "binary-name"
           PROVENANCE: ${{ needs.build.outputs.provenance-name }}
         run: ./.github/workflows/scripts/pre-submit.e2e.generic.default.sh
+
+  build-continue-error:
```

Comment:
We probably want a pre-submit for the container workflow too since we are trying to GA it.

Reply:
Ah, I forgot we can't push to ghcr.io on pull_request. We'll need to make an e2e test.